bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -600,12 +600,16 @@ public boolean isAborted() {
 
   @Override
   public void truncateTable(TableName tableName, boolean preserveSplits) throws IOException {
-    throw new UnsupportedOperationException("truncateTable");  // TODO
+    HTableDescriptor tableDescriptor = getTableDescriptor(tableName);
+    LOG.info("Deleting table %s", tableName);
```

Comment:
this log could be mislead, makes someone think that the table is being delete.
how about "Deleting table %s for truncation"?